From 267967e1d77d9da490bf4bfbdbfaf36d3e5805b6 Mon Sep 17 00:00:00 2001
From: Manorit Chawdhry <m-chawdhry@ti.com>
Date: Wed, 29 Nov 2023 18:02:24 +0530
Subject: [PATCH] binman: j721s2: Remove firewall ID 5143, 5144

The following firewall is not acting correctly and creating problems
with CSI capture, remove the configuration of the firewall till the
issue is root caused.

Fix: LCPD-37067

Signed-off-by: Manorit Chawdhry <m-chawdhry@ti.com>
---
 arch/arm/dts/k3-j721s2-binman.dtsi | 57 ------------------------------
 1 file changed, 57 deletions(-)

diff --git a/arch/arm/dts/k3-j721s2-binman.dtsi b/arch/arm/dts/k3-j721s2-binman.dtsi
index 72278f83227a..ae4019c1cfe7 100644
--- a/arch/arm/dts/k3-j721s2-binman.dtsi
+++ b/arch/arm/dts/k3-j721s2-binman.dtsi
@@ -347,63 +347,6 @@
 							start_address = <0x0 0x9e800000>;
 							end_address = <0x0 0x9fffffff>;
 						};
-
-						firewall-5143-0 {
-							/* nb_slv3__mem0 Background Firewall - 0 */
-							id = <5143>;
-							region = <0>;
-							control = <(FWCTRL_EN | FWCTRL_LOCK |
-										FWCTRL_BG | FWCTRL_CACHE)>;
-							permissions = <((FWPRIVID_ALL << FWPRIVID_SHIFT) |
-											FWPERM_SECURE_PRIV_RWCD |
-											FWPERM_SECURE_USER_RWCD |
-											FWPERM_NON_SECURE_PRIV_RWCD |
-											FWPERM_NON_SECURE_USER_RWCD)>;
-							start_address = <0x0 0x0>;
-							end_address = <0xff 0xffffffff>;
-						};
-
-						firewall-5143-1 {
-							/* nb_slv3__mem0 Foreground Firewall */
-							id = <5143>;
-							region = <1>;
-							control = <(FWCTRL_EN | FWCTRL_LOCK |
-										FWCTRL_CACHE)>;
-							permissions = <((FWPRIVID_ARMV8 << FWPRIVID_SHIFT) |
-											FWPERM_SECURE_PRIV_RWCD |
-											FWPERM_SECURE_USER_RWCD)>;
-							start_address = <0x0 0x9e800000>;
-							end_address = <0x0 0x9fffffff>;
-						};
-
-						firewall-5144-0 {
-							/* nb_slv4__mem0 Background Firewall - 0 */
-							id = <5144>;
-							region = <0>;
-							control = <(FWCTRL_EN | FWCTRL_LOCK |
-										FWCTRL_BG | FWCTRL_CACHE)>;
-							permissions = <((FWPRIVID_ALL << FWPRIVID_SHIFT) |
-											FWPERM_SECURE_PRIV_RWCD |
-											FWPERM_SECURE_USER_RWCD |
-											FWPERM_NON_SECURE_PRIV_RWCD |
-											FWPERM_NON_SECURE_USER_RWCD)>;
-							start_address = <0x0 0x0>;
-							end_address = <0xff 0xffffffff>;
-						};
-
-						firewall-5144-1 {
-							/* nb_slv4__mem0 Foreground Firewall */
-							id = <5144>;
-							region = <1>;
-							control = <(FWCTRL_EN | FWCTRL_LOCK |
-										FWCTRL_CACHE)>;
-							permissions = <((FWPRIVID_ARMV8 << FWPRIVID_SHIFT) |
-											FWPERM_SECURE_PRIV_RWCD |
-											FWPERM_SECURE_USER_RWCD)>;
-							start_address = <0x0 0x9e800000>;
-							end_address = <0x0 0x9fffffff>;
-						};
-
 					};
 					tee: tee-os {
 					};
-- 
2.41.0

